frontend/device/devicelistener.lua

```diff
@@ -9,9 +9,7 @@ local bit = require("bit")
 local _ = require("gettext")
 local T = require("ffi/util").template
 
-local DeviceListener = InputContainer:new{
-    steps_fl = { 0.1, 0.1, 0.2, 0.4, 0.7, 1.1, 1.6, 2.2, 2.9, 3.7, 4.6, 5.6, 6.7, 7.9, 9.2, 10.6, },
-}
+local DeviceListener = InputContainer:new{}
```

Comment:
I'm fine with this, but it's not necessarily what I was saying.

```lua
-- all that's strictly necessary
local DeviceListener = InputContainer:new{}
```

```lua
-- reduced diff if you add something in the future
local DeviceListener = InputContainer:new{
}
```

```lua
-- random new line, that'll be two lines on the diff rather than one if you add something
local DeviceListener = InputContainer:new{

}
```

Reply:
OK, I don't want to add anything.